build/commands/lib/buildChromium.js

```diff
@@ -0,0 +1,157 @@
+    return config.getTargetOS()
+  })()
+  return `chromium-${config.chromeVersion}-${platform}-${config.targetArch}`
+}
+
+const chromiumConfigs = {
+  'win': {
+    buildTarget: 'mini_installer',
+    processArtifacts: () => {
+      // Repack it to reduce the size and use .zip instead of .7z.
+      input = path.join(config.outputDir, 'chrome.7z')
+      output = path.join(config.outputDir, `${getOutputFilename()}.zip`)
+      util.run('python3',
+        [
+          path.join(config.braveCoreDir, 'script', 'repack-archive.py'),
+          `--input=${input}`,
+          `--output=${output}`,
+          '--target_dir=Chrome-bin',
+        ],
+        config.defaultOptions)
+    }
+  },
+  'linux': {
+    buildTarget: 'chrome/installer/linux:stable_deb',
+    processArtifacts: () => {
+      const debArch = (() => {
+        if (config.targetArch === 'x64') return 'amd64'
+        return config.targetArch
+      })()
+      fs.moveSync(
+        path.join(config.outputDir,
+          `chromium-browser-stable_${config.chromeVersion}-1_${debArch}.deb`),
+        path.join(config.outputDir, `${getOutputFilename()}.deb`))
+    }
+  },
+  'mac': {
+    buildTarget: 'chrome',
+    processArtifacts: () => {
+      util.run('zip',
+        ['-r', '-y', `${getOutputFilename()}.zip`, 'Chromium.app'],
+        { cwd: config.outputDir }
+      )
+    }
+  },
+  'android': {
+    buildTarget: 'chrome_public_apk',
+    processArtifacts: () => {
+      fs.moveSync(
+        path.join(config.outputDir, 'apks', 'ChromePublic.apk'),
+        path.join(config.outputDir, `${getOutputFilename()}.apk`))
+    }
+  },
+}
+
+// A function to make gn args to build a release Chromium build.
+// There is two primarily sources:
+// 1. Chromium perf builds: tools/mb/mb_config_expectations/chromium.perf.json
+// 2. Brave Release build configuration
+function getChromiumGnArgs() {
+  const braveGnArgs = config.buildArgs()
+  const chromiumGnArgs = {
+    target_cpu: config.targetArch,
+    target_os: config.getTargetOS(),
+    is_official_build: true,
+    symbol_level: 1,
+    enable_keystone_registration_framework: false,
+  }
+
+  if (config.isAndroid)
+    chromiumGnArgs.debuggable_apks = false
+
+  const gnArgsToMirror = [
+    'enable_hangout_services_extension',
+    'enable_widevine',
+    'ignore_missing_widevine_signing_cert',
+    'enable_nacl',
+    'ffmpeg_branding',
+  ]
+
+  if (braveGnArgs.use_system_xcode !== undefined) {
+    chromiumGnArgs.use_system_xcode = braveGnArgs.use_system_xcode
+  }
+
+  for (const arg of gnArgsToMirror) {
+    const braveArg = braveGnArgs[arg]
+    if (braveArg == undefined)
+      throw Error(`Gn arg ${arg} doesn't exist in config.buildArgs()`)
+    chromiumGnArgs[arg] = braveArg
+  }
+  return chromiumGnArgs
+}
+
+function buildChromium(buildConfig = config.defaultBuildConfig, options = {}) {
+  config.buildConfig = buildConfig
+  config.update(options)
+  config.outputDir = config.outputDir + '_chromium'
```

Comment:
We have `-C` parameter which can be used to set an explicit output directory without any modifications. With yoru implementation, `_chromium` will always be appended to `-C` value.

can we introduce a parameter like `isChromium` or `outputDirSuffix` into `Config` and set it in `config.update(options)` call in the previous line? This parameter can then be used [here](https://github.com/brave/brave-core/blob/239c2ccf489b3c89ecec54d9e0b1a9a53160216d/build/commands/lib/config.js#L1292C1-L1292C1) to modify `outputDir` if it's not set via `-C` explicitly.

Reply:
Added `isChromium`. Also removed `-C` support, because we don't support building anything except Chromium Release right now.

Reply:
> Added `isChromium`. Also removed `-C` support, because we don't support building anything except Chromium Release right now.

why? `-C` is a very cool feature, let's keep it. You can build the same configuration in multiple directories, that's sometimes is usable.